services/application.js

```diff
@@ -370,8 +368,6 @@ class Application {
         }else{
             cc_email = admin_email
         }
-        // submission documentation 
```

Comment:
after removing this line, the code fails on line 374 because sub_doc_url is undefined